process-manager.js

```diff
@@ -142,7 +140,10 @@ class ProcessManager {
 }
 
 ProcessManager.cache = processManagers;
-module.exports = ProcessManager;
+module.exports = {
+	ProcessManager: ProcessManager,
+	ProcessWrapper: ProcessWrapper,
```

Comment:
It's a design decision that directly exposing the wrapper shouldn't be needed.

Changing that like this needlessly complicates creating a PM.

If you need it for testing, maybe we can live with ProcessManager.wrapperClass = ProcessWrapper or something like that?